workspaces/analytics/.changeset/neat-monkeys-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-analytics-module-matomo': major
```

Comment:
Do you really want this to be major? Will get releases as `1.1.0`

Reply:
Switched to minor, also it has Changelog.md so I have set the package version to that of janus idp